### PR TITLE
Update MainResources-local-openapi-v3.yml

### PR DESCRIPTION
**Pull request by SteffenMangold**
Add missing open door/window window element

### CODE REVIEW

File: openapi/MainResources-local-openapi-v3.yml [hunk 1]
```diff
@@ -464,31 +462,31 @@ components:
...
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         unknownWindows:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         allOthers:
```
Comment (sebastianharder) [inline openapi/MainResources-local-openapi-v3.yml:486]: `allOthers` should not be available anymore

File: openapi/MainResources-local-openapi-v3.yml [hunk 2]
```diff
@@ -464,31 +462,31 @@ components:
...
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'  
         unknownDoors:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         allWindows:
```
Comment (sebastianharder) [inline openapi/MainResources-local-openapi-v3.yml:474]: `allWindows` should not be available anymore

File: openapi/MainResources-local-openapi-v3.yml [hunk 3]
```diff
@@ -464,31 +462,31 @@ components:
         allDoors:
```
Comment (sebastianharder) [inline openapi/MainResources-local-openapi-v3.yml:462]: `allDoors` should not be available anymore

File: openapi/MainResources-local-openapi-v3.yml (hunk 2 above)
Comment (SteffenMangold) [reply openapi/MainResources-local-openapi-v3.yml:474]: Currently it still is
image

File: openapi/MainResources-local-openapi-v3.yml (hunk 1 above)
Comment (SteffenMangold) [reply openapi/MainResources-local-openapi-v3.yml:486]: On HomeController I (newest version) it still is
image

File: openapi/MainResources-local-openapi-v3.yml (hunk 2 above)
Comment (sebastianharder) [reply openapi/MainResources-local-openapi-v3.yml:474]: Oh right, I just remembered that we had removed it from the push notification that is sent to Android due to size restrictions. Thought, we removed it for the REST endpoint as well.

**Review by sebastianharder — APPROVED** [review]

**Pull request merged**